install_config/install/advanced_install.adoc

```diff
@@ -589,6 +595,84 @@ openshift_master_overwrite_named_certificates=true
 ----
 ====
 
+[[advanced-install-cluster-metrics]]
+== Configuring Cluster Metrics
+
+Cluster metrics are not set to automatically deploy by default. Set the
+following to enable cluster metrics when using the advanced install:
+
+====
+----
+[OSEv3:vars]
+
+openshift_hosted_metrics_deploy=true
+----
+====
+
```

Comment:
It ties into #2763 I'll reassess links etc when both this and that one are merged. (I have made a note to myself.)